### ISSUE TITLE
No Cuda?

### ISSUE THREAD

**Issue by lucellent** [bytedance/uss#3]
Is there an option to inference with GPU, as currently I'm only using the CPU

**Comment by lucellent** [comment]
Also, are there any overlap features or something similar to improve predictions, since the outputs have clicks to them

**Comment by qiuqiangkong** [comment]
Yes, the program should run with CPU. Please let us know if there are any
problems.
